packages/frontend/src/embed/EmbedApp.vue

```diff
@@ -7,6 +7,8 @@ export default {
   mounted() {
     let mixpanelId = this.$mixpanelId()
     this.$mixpanel.register({
+      // Unfortunately we can't replace this to camelCase, because that will break metrics
+      // eslint-disable-next-line camelcase
```

Comment:
Not great as we can't really change the property. We could create a helper/mixin that wraps this call, however, so that there's only 1 line that needs to be ignored.